src/analyze/analyze-plot.c

```diff
@@ -130,6 +138,12 @@ static int compare_unit_start(const UnitTimes *a, const UnitTimes *b) {
 }
 
 static void svg_graph_box(double height, double begin, double end) {
+        /* show timestamps relative to the beginning of the plot
+         * keep the firmware and loader timestamps at the negative side */
+        if (begin > 0) {
+                end = end - begin;
```

Comment:
`usec_sub_unsigned()` ?

Reply:
No, `begin` and `end` are both `double` and the former is expected to be negative which won't work with unsigned `usec_t`.

Reply:
Oh, indeed. Sorry for the noise.